src/sentry/identity/base.py

```diff
@@ -1,25 +1,10 @@
 from __future__ import absolute_import, print_function
 
 import logging
-from collections import namedtuple
 
 from sentry.pipeline import PipelineProvider
 
 
-class MigratingIdentityId(namedtuple("MigratingIdentityId", ["id", "legacy_id"])):
```

Comment:
Removed this because it wasn't being used, and the instanceof checks would be wrong when we check `MigratingIdentitites` for the newstyle google IDP against oldstyle `AuthProvider`s.